### ISSUE TITLE
The "Tap targets are not sized appropriately" metric should be under Accessibility, not SEO

### ISSUE THREAD

**Issue by proimage** [GoogleChrome/web.dev#5053]
**Describe the bug**
The "Tap targets are not sized appropriately" metric should be under Accessibility, not SEO.

**To Reproduce**
1. Test any site with too-small tap targets.

**Expected behavior**
I expected to find "Tap targets are not sized appropriately" categorized as an Accessibility metric, not an SEO one.

**Screenshots**
![image](https://user-images.githubusercontent.com/1027523/114158990-fcdf4200-992d-11eb-9955-eed0e2f60724.png)
